apps/web/app/layout.tsx

```diff
@@ -69,11 +67,13 @@ export default async function RootLayout({ children }: { children: React.ReactNo
       style={embedColorScheme ? { colorScheme: embedColorScheme as string } : undefined}
       data-nextjs-router="app">
       <head nonce={nonce}>
-        {!IS_PRODUCTION && process.env.VERCEL_ENV === "preview" && (
-          // eslint-disable-next-line @next/next/no-sync-scripts
-          <Script
-            data-project-id="KjpMrKTnXquJVKfeqmjdTffVPf1a6Unw2LZ58iE4"
-            src="https://snippet.meticulous.ai/v1/stagingMeticulousSnippet.js"
+        {process.env.NEXT_PUBLIC_HEAD_SCRIPTS && (
```

Comment:
Tiny NIT: Probably a good idea to do the following as otherwise if "1" is injected as a string- it'll potentially show up as a string value here due to jsx.

```suggestion
        {!!process.env.NEXT_PUBLIC_HEAD_SCRIPTS && (
```